app/assets/stylesheets/components/form.scss

```diff
@@ -6,3 +6,35 @@ input.btn.btn-default.btn-success.btn-block {
   font-weight: bold;
   font-size: $base-font-size;
 }
+
+.form-section {
+	background-color: $white;
```

Comment:
Sublime Text > Preferences > Settings (if you want it to always use it)
Sublime Text > Preferences > Settings - Syntax Specific (while the file type to want to change is currently open, if you want it to always use it when using a particular file-type)

```
{
	"tab_size": 2,
	"translate_tabs_to_spaces": true,
	"trim_trailing_white_space_on_save": true,
}
```